governance/voter-weight-addin/program/src/instruction.rs

```diff
@@ -2,53 +2,48 @@
 
 use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
 use solana_program::{
+    clock::Slot,
     instruction::{AccountMeta, Instruction},
     pubkey::Pubkey,
     system_program,
 };
+use spl_governance::addins::voter_weight::VoterWeightAction;
 
 /// Instructions supported by the VoterWeightInstruction addin program
 #[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
 #[allow(clippy::large_enum_variant)]
 pub enum VoterWeightAddinInstruction {
-    /// Revises voter weight providing up to date voter weight
+    /// Sets up VoterWeightRecord owned by the program
     ///
     /// 0. `[]` Governance Program Id
     /// 1. `[]` Realm account
     /// 2. `[]` Governing Token mint
     /// 3. `[]` TokenOwnerRecord
     /// 4. `[writable]` VoterWeightRecord
-    Revise {},
```

Comment:
I imagine that it's ok to break this interface?

Reply:
yes, the `addin` is just a programable mock program used for testing only